src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
well, something like that may solve things for FrameworkBundle 2.8 users (it needs to be a bit more complex though, as you can use different versions of FrameworkBundle and Validator). but it is still a BC break:

- someone running FrameworkBundle 2.7 with the new validator (and this is allowed by FrameworkBundle 2.8 thanks to our BC promise) would get broken things
- someone using another framework (Silex for instance) and registering a service a service for the expression validator would get broken things